layouts/partials/tw/main-elections.html

```diff
@@ -185,10 +270,10 @@
     ".PublishDate.Unix" "gt" $primary
   }}
   {{ partial "tw/list-3-col-story.html" (dict
-    "hed" "Elecciones PA 2022: Traducciones al Español"
+    "hed" "Elecciones Pa. 2023: Traducciones al Español"
```

Comment:
Ask Sarah if she wants "PA" or "Pa." here.

Reply:
It was "Pa." in the google doc, but will double check.